src/Order.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable indent*/
+import React, { useState } from "react";
+import RestaurantButton from "./RestaurantButton";
+
+function Order(props) {
+  const [orders, setOrder] = useState(0);
+  function addOne() {
+    const newQuantity = orders + 1;
+    setOrder(newQuantity);
+  }
+  return (
+    <li className="order-list">
+      {props.orderType}:{orders}
+      <RestaurantButton update={addOne} />
+    </li>
+  );
+}
```

Comment:
This is a great use of props to create a reusable component, to make this even better, have you tried adding 1 to the order number within the setOrder() function? This would remove the need for the newQuantity constant completely!